core/src/main/java/com/amplifyframework/api/RestBehavior.java

```diff
@@ -31,222 +31,258 @@
     /**
      * This is a helper method for easily invoking GET HTTP request.
      * Requires that only one API is configured in your
-     * `amplifyconfiguration.json`. Otherwise, emits an ApiException
-     * to the provided `ResultListener`.
+     * `amplifyconfiguration.json`. Otherwise, emits an {@link ApiCategory}
```

Comment:
Should be ApiException vs. ApiCategory

Reply:
Whoops, nice catch, thanks - updating, one sec...